### DISCUSSION TITLE
Tunable title page for slides

### DISCUSSION THREAD

**Discussion by awav** [quarto-dev/quarto-cli#2948] [Q&A]
Hi everyone,

It would be extremely useful to be able to modify the title page for slides, particularly in revealjs. Ultimately, I would like to add pictures to the title page, and it would be good to have animation feature as well.

Also, I'm a bit confused, why it doesn't have a similar template as [Title Block](https://quarto.org/docs/authoring/title-blocks.html). Basically, right now it is a pile of text with different sizes placed on top of each other.

**Comment by dragonstyle** [comment]
It was merely an issue of running out of time - our pre-release of version 1.2 includes support for this - you may provide partials for `title-slide.html` or `toc-slide.html`.

You can get the latest pre-release here:
https://quarto.org/docs/download/prerelease.html

(Also just realized we do have some limited docs on this here: https://quarto.org/docs/presentations/revealjs/advanced.html#custom-template)

**Reply by awav** [reply]
Thanks! I have seen those options, and it is great to have some ways to adapt the first page. However, this is a bit unfair to users. Not all quarto users will be experienced in the web development. Even one or two examples in the documentation on how to update those files or make them similar to the [Tile Block](https://quarto.org/docs/authoring/title-blocks.html) would be extremely helpful.

**Reply by dragonstyle** [reply]
Yeah I agree it would be even better if there were additional title-block styles for revealjs presentations- this is something we'll definitely try to get to in the future. 

In the meantime, if there are any motivated individuals who do want to provide a custom presentation, Quarto also supports extensions for formats which would allow someone to package up a revealjs template with a title slide, theme, and so on into an improved version that is easy for users to use...

**Reply by awav** [reply]
Also, quarto has the gallery with advanced examples of slides: <https://quarto.org/docs/gallery/>.  The problem is that there are no links to the source code. I found the source code for one of the examples after some google and github search: <https://github.com/emitanaka/talks/blob/master/Toronto2022/toronto2022.qmd>.

**Reply by mcanouil** [reply]
I would suggest to take a look at <https://quarto.org/docs/extensions/listing-revealjs.html> and <https://github.com/mcanouil/awesome-quarto#presentations> to get some title page example for Reveal.js.

Did you not see the code source button on the top right corner?
<img width="1076" alt="image" src="https://user-images.githubusercontent.com/8896044/197036018-559d5dad-b08f-44a8-8cbf-d36119e12f9e.png">

**Reply by awav** [reply]
@mcanouil thanks a lot. Indeed, I've missed the source code link...

**Reply by mcanouil** [reply]
You're welcome 😉
